yellowbrick/contrib/wrapper.py

```diff
@@ -0,0 +1,134 @@
+# yellowbrick.contrib.wrapper
+# Wrapper for third-party estimators that implement the sklearn API
+#
+# Author:   Benjamin Bengfort
+# Created:  Fri Oct 02 14:47:54 2020 -0400
+#
+# Copyright (C) 2020 The scikit-yb developers
+# For license information, see LICENSE.txt
+#
+# ID: wrapper.py [] benjamin@bengfort.com $
+
+"""
+Wrapper for third-party estimators that implement the sklearn API but do not directly
+subclass the ``sklearn.base.BaseEstimator`` class. This method is a quick way to get
+other estimators into Yellowbrick, while avoiding weird errors and issues.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+from yellowbrick.exceptions import YellowbrickAttributeError
+
+
+##########################################################################
+## Module Constants
+##########################################################################
+
+CLASSIFIER = "classifier"
+REGRESSOR = "regressor"
+CLUSTERER = "clusterer"
+DENSITY_ESTIMATOR = "DensityEstimator"
+OUTLIER_DETECTOR = "outlier_detector"
```

Comment:
I pulled these from the [scikit-learn source code](https://github.com/scikit-learn/scikit-learn/blob/0fb307bf3/sklearn/base.py) - we don't have density estimator or outlier detector visualizer … but hopefully some day.